app/src/main/java/de/danoeh/antennapod/activity/PreferenceActivity.java

```diff
@@ -141,9 +142,12 @@ public boolean onOptionsItemSelected(MenuItem item) {
                 finish();
             } else {
                 InputMethodManager imm = (InputMethodManager) getSystemService(INPUT_METHOD_SERVICE);
-                if (imm.isActive()) {
-                    imm.toggleSoftInput(InputMethodManager.SHOW_IMPLICIT, InputMethodManager.HIDE_NOT_ALWAYS);
+                View view = getCurrentFocus();
+                //If no view currently has focus, create a new one, just so we can grab a window token from it
+                if (view == null) {
+                    view = new View(this);
                 }
+                imm.hideSoftInputFromWindow(view.getWindowToken(), 0);
```

Comment:
Apparently, something went wrong while rebasing. This change (and many of the others) should not be displayed, as they already are on `develop`. Did you rebase onto an old version of the branch?

Reply:
Yeah, tried to resolve it today. For some reason, 'rebase --onto' isn't working. That's what I've used in the past and it worked. Maybe I doing wrong this time. Will try later today.